### PR TITLE
Add Wordy Exercise to Wren

### PR DESCRIPTION
**Pull request by joshgoebel**
- wip
- wip
- wip

### CODE REVIEW

**Comment by joshgoebel** [comment]
@glennj Want to have a look?

File: exercises/practice/wordy/wordy.wren
Comment (glennj) [inline exercises/practice/wordy/wordy.wren:1]: I thought we were getting rid of these comment headers in the skeletons.

**Comment by glennj** [comment]
Looks reasonable. Please take a formatting pass over the proof.

File: exercises/practice/wordy/wordy.wren
Comment (joshgoebel) [reply exercises/practice/wordy/wordy.wren:1]: They are still in JS.  Removed.

**Comment by joshgoebel** [comment]
> Please take a formatting pass over the proof.

Something in particular?  Looks good to me.

**Comment by glennj** [comment]
line 17, indentation,  
line 19, no space around operator,  
no space after comma,    
inconsistent newline before returns,

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
I can't merge this, no button. @iHiD can I be given merge privs in this repo?

**Comment by joshgoebel** [comment]
@glennj I actually don't intend you to LOL, sorry if that's a miscommunication, sometimes I approve things then come back a day later and take one final look.  Yeah, I'm weird.

I think I'd be fine if you had merge bits though if @iHiD is fine with that, I dunno if that then makes you a maintainer (or something else)... I don't know the politics of it all.  I sure do appreciate the contributions you've made to Wren though. :-)  Hopefully they keep coming. :-)

**Pull request merged**
